README.md

```
@@ -1,2 +1,112 @@
+| Local DB | DataStore (Preferences) |
+| Navigation | Compose Navigation |
+
+**클린 아키텍쳐 선정 이유**
+
+클린 아키텍쳐는 도메인 레이어를 두어 데이터와 UI를 분리합니다.
+도메인 레이어는 코틀린 언어로 작성되었기 때문에 데이터 또는 UI 레이어에 사용되는 외부 도구에 문제가 생기더라도 순수하게 보존됩니다.
+구글 권장 아키텍쳐에 비해 코드 작성량이 많다는 단점이 있으나, 엄격한 의존성 분리와 정형화된 프로젝트 구조를 유도합니다.
+
+<br>
+
+**MVI 패턴 선정 이유**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)